Detectors/ITSMFT/ITS/tracking/include/ITStracking/Definitions.h

```diff
@@ -21,7 +21,7 @@
 #include <array>
 #endif
 
-//#define CA_DEBUG
+#define CA_DEBUG
```

Comment:
Better have it commented in dev, this will slow down standard reco.

Reply:
Sure, it was just a demo for the draft pr.